Components/Input/input.css

```diff
@@ -21,19 +20,15 @@ input[type="password"] {
 }
 
 #text-input:focus {
-  border: 3px solid var(--primary);
+  border: 2px solid var(--primary-dark);
 }
 
 #success-input {
-  border: 3px solid var(--success);
+  border: 2px solid var(--success);
 }
 
 #error-input {
-  border: 3px solid var(--danger);
-}
-
-.gap-0 {
-  gap: 0;
+  border: 2px solid var(--danger);
```

Comment:
Suggestion: Avoid using Id selectors until it's really necessary as they have very high specificity.

Reply:
Yes, for now i've kept it that way. Will add a proper class and fix it.